packages/pageheader/README.md

```diff
@@ -0,0 +1,19 @@
+# PageHeader Name
+
+PageHeader documentation
+
+Example Implementation
+
+```JSX
```

Comment:
What's the thinking behind adding example implementation in the README? Storybook can do that automatically.